common/text-level-semantics/text-level-semantics.html

```diff
@@ -0,0 +1,148 @@
+---
+{
+	"title": "Text level semantics",
+	"language": "en",
+	"altLangPage": "texte-niveau-semantique.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-07-06"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<div class="alert alert-warning">
+<p>The purpose of this page is to test all native content grouping related elements, if they are aligned with our design and are compliant to our accessibility guideline when used as is without any special customization. The following include all content grouping elements in the HTML5 specification and a few examples was inspired by the <a href="https://html.spec.whatwg.org/multipage/forms.html#forms">WHATWG section 4.4</a> as February 2023. This page may not contain all the possible grouping content element combination.</p>
```

Comment:
Please update this link + text link to match the content in the HTML5 specification

```suggestion
<p>The purpose of this page is to test all native content grouping related elements, if they are aligned with our design and are compliant to our accessibility guideline when used as is without any special customization. The following include all content grouping elements in the HTML5 specification and a few examples was inspired by the <a href="https://html.spec.whatwg.org/multipage/text-level-semantics.html#text-level-semanticss">WHATWG section 4.5</a> as September 2023. This page may not contain all the possible grouping content element combination.</p>
```